examples/flutter_gallery/ios/Runner/AppDelegate.m

```diff
@@ -0,0 +1,6 @@
+#import <Flutter/Flutter.h>
+#include "AppDelegate.h"
+
+@implementation AppDelegate
+
```

Comment:
`flutter create` will generate a bunch of stub methods with docs, but I haven't for the samples.